test_scripts/API/ButtonSubscription/031_SubscribeButton_on_different_buttons_Unexpected_disconect.lua

```diff
@@ -0,0 +1,67 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0192-button_subscription_response_from_hmi.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1) Mobile app is subscribed for button_1 and button_2
+-- 2) Unexpected disconnect and connect are performed
+-- 3) App registers with actual hashId
+-- 4) SDL sends Buttons.SubscribeButton(button_1, appId) to HMI during resumption
+-- 5) SDL sends Buttons.SubscribeButton(button_2, appId) to HMI during resumption
+-- 6) HMI responds with error code to SubscribeButton(button_1, appId)
+-- SDL does:
+-- 1) Process error response from HMI and revert subscription for button_2
+-- 2) Respond RAI(RESUME_FAILED) to mobile app
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local utils = require("user_modules/utils")
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/ButtonSubscription/common_buttonSubscription')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local variable ]]
+local buttonName_1 = "PRESET_0"
+local buttonName_2 = "PRESET_1"
+
+--[[ Local Functions ]]
+local function checkResumptionData(pAppId)
+    EXPECT_HMICALL("Buttons.SubscribeButton",
+        { appID = common.getHMIAppId(pAppId), buttonName = "CUSTOM_BUTTON" },
+        { appID = common.getHMIAppId(pAppId), buttonName = "PRESET_0" },
+        { appID = common.getHMIAppId(pAppId), buttonName = "PRESET_1" })
+    :Times(3)
+    :Do(function(_, data)
+        if data.params.buttonName == "PRESET_0" then 
+            common.getHMIConnection():SendError(data.id, data.method, "GENERIC_ERROR", { })
```

Comment:
SendError function has another parameters, 4th parameter must be string:
`
SendError(data.id, data.method, "GENERIC_ERROR", "Error message")
`